app/src/main/java/org/simple/clinic/splash/SplashScreen.kt

```diff
@@ -5,25 +5,34 @@ import android.util.AttributeSet
 import androidx.constraintlayout.widget.ConstraintLayout
 import androidx.navigation.findNavController
 import com.airbnb.lottie.LottieDrawable
-import kotlinx.android.synthetic.main.screen_splash.view.*
 import org.simple.clinic.R
+import org.simple.clinic.databinding.ScreenSplashBinding
 
 class SplashScreen(context: Context, attributeSet: AttributeSet) : ConstraintLayout(context, attributeSet) {
 
+  private var _binding: ScreenSplashBinding? = null
+  private val binding get() = _binding!!
```

Comment:
Maybe we can add accessor properties for the child views instead of the `binding` property?

```kotlin
private val splashLottieView
  get() = binding!!.splashLottieView
```

Reply:
In this way, we will not need to change the event setup, just add whatever views we need to reference in one place.

Reply:
Oh yeah, good idea. I was just going with the default view binding setup, will change it.

Reply:
Done